### ISSUE TITLE
Does not compile on Android arm

### ISSUE THREAD

**Issue by dmerejkowsky** [eminence/terminal-size#10]
Steps to reproduce:

```ini
# In ~/.cargo/config
[target.armv7-linux-androideabi]
linker = "path/to//android-arm-api-27-toolchain/bin/arm-linux-androideabi-gcc"
```
```
$ cargo build
/home/dmerej/.cargo/registry/src/github.com-1ecc6299db9ec823/terminal_size-0.1.8/src/unix.rs:33:30
   |
33 |         ioctl(STDOUT_FILENO, TIOCGWINSZ, &mut winsize);
   |                              ^^^^^^^^^^ expected i32, found u32

```

I think there's a `#[cfg]` missing somewhere

_Update_ For now I edit the file by hand to add `TIOCGWINSZ as i32` but this is not ideal ...

As to why I need terminal size on android, well, it's a long story :)

**Comment by dmerejkowsky** [comment]
By the way, [term_size](https://crates.io/crates/term_size) does not have the problem, so I'm using this instead now.

**Comment by TimonPost** [comment]
Thanks will take a look at! But yeah with cross-platform I did not have Android in my mind ;) but I'll see what I can do.

**Comment by pawamoy** [comment]
> As to why I need terminal size on android, well, it's a long story :)

Or a short one! I want to try [termpix](https://github.com/hopey-dishwasher/termpix) in Termux! And terminal-size is a dependency! That's it 😄

**Comment by eminence** [comment]
This build error should be fixed when #14 is merged
